server/controllers/ssotRetrievalController.js

```diff
@@ -168,3 +168,28 @@ exports.createNewRobot = async (req, res) => {
     console.error(err);
   }
 };
+
+// POST /overwriteRobot/78d09f66d2ed466cf20b06f7
+exports.overwriteRobot = async (req, res) => {
+  try {
+    res.set('Content-Type', 'application/json');
+    const updatedSsot = req.body;
+    
+    const ssotData = await mongoose
+      .model('SSoT')
+      .findByIdAndUpdate(
+        updatedSsot['_id'], 
```

Comment:
EsLint wants dot notation

Reply:
is fixed

Reply:
Will be changed back by me. Using dot notation there has a side-effect of causing Mongoose to try upserting the Id which causes an error and prevents the local data to be upserted to the database. @lukashueller @WolfgangDaniel

Reply:
okay